sdk/core/core/test/mocka_json.c

```diff
@@ -0,0 +1,62 @@
+#include <stdarg.h>
+#include <stddef.h>
+
+#include <setjmp.h>
+#include <stdint.h>
+
+#include <az_json.h>
+#include <az_span.h>
+
+#include <cmocka.h>
+
+static void test_json_token_null(void ** state) {
+  (void)state; /* unused */
+  az_json_token token = az_json_token_null();
+  assert_int_equal(token.kind, AZ_JSON_TOKEN_NULL);
+  assert_false(token.value.boolean);
+  assert_int_equal(token.value.number, 0);
+
+  assert_ptr_equal(az_span_ptr(token.value.string), NULL);
+  assert_int_equal(az_span_capacity(token.value.string), 0);
+  assert_int_equal(az_span_length(token.value.string), 0);
+
+  assert_ptr_equal(az_span_ptr(token.value.span), NULL);
+  assert_int_equal(az_span_capacity(token.value.span), 0);
+  assert_int_equal(az_span_length(token.value.span), 0);
+}
+
+static void test_json_token_boolean(void ** state) {
+  (void)state; /* unused */
```

Comment:
`/* unused */` comment is redundant, in my opinion. If you know the language, there's really no other explanation to the `(void)state; line other than it is to clearly indicate that it is not used, but suppress warning.